core/modules/savers/put.js

```diff
@@ -15,6 +15,24 @@ to the current URL, such as a WebDAV server.
 /*global $tw: false */
 "use strict";
 
+/*
+Retrieve ETag if available
+*/
+var RetrieveETag = function(self) {
+	var headers = { "Accept": "*/*;charset=UTF-8" };
```

Comment:
needs to be tested.

Reply:
You are right. If we use `var headers = {}` my FF browser uses `Accept: */*` by default. ... The `charset=` is probably a copy paste problem from `.save()`

Reply:
So if you want to make it more readable we should stay with the accept header. Otherwise users may be curious why it is there at all. ...  

	var headers = { "Accept": "*/* };